coiote-aws-iot-cloud-formation/createThing/create_thing.py

```diff
@@ -13,7 +13,7 @@ def ensure_thing_type_exists(thing_type):
 
 
 def lambda_handler(event, context):
-    thing_name = event['coioteDeviceId']
+    thing_name = event['coioteDeviceId'].replace('%3A', ':')
     thing_type_name = event['coioteDeviceType'].replace('.', ':')
```

Comment:
Extract these 2 to separate method and `%3A` to const. Add short doc to this method explaining why we need such escaping